src/send/send_caliper_events.py

```diff
@@ -0,0 +1,69 @@
+import requests
+import fileinput
```

Comment:
The program never uses this imported module, `fileinput`.  If you use the code clean-up feature of your IDE, it will probably detect and remove unused imports for you.